### PR TITLE
gyp: Remove duplicate items in gyp files and update msvs.py

### PR DESCRIPTION
**Pull request by munyirik**
* There were a couple of unnecessary duplicates between common.gypi,
node.gyp, and addon.gypi that I've removed.
* Updated msvs.py to match my commit in the chromium gyp branch
(https://codereview.chromium.org/1946883002)

### CODE REVIEW

**Comment by munyirik** [comment]
@kunalspathak could you take a look? Normally I'd get updates to gyp from chakracore but I need this to fix errors that occur when building native addons targeting UWP.

**Comment by munyirik** [comment]
Also need to update https://github.com/ms-iot/node/blob/chakra-uwp/common.gypi#L14

File: common.gypi [hunk 1]
```diff
@@ -77,11 +77,7 @@
         'defines': [
           'NODE_ENGINE_CHAKRA',
         ],
-        'conditions': [
-          ['node_engine=="chakra" or target_arch=="arm"', {
-            'msvs_windows_sdk_version': '<(msvs_windows_sdk_version)',
-          }],
-        ],
+        'msvs_windows_sdk_version': '<(msvs_windows_sdk_version)',
```
Comment (kunalspathak) [inline common.gypi]: why is condition for `chakra` removed here?

**Comment by kunalspathak** [comment]
:shipit:

File: common.gypi (hunk 1 above)
Comment (munyirik) [inline common.gypi]: the condition is already there in line 75 so it wasn't necessary.

**Pull request merged**